### ISSUE TITLE
Do this project support nn.ConvTranspose2d layer?

### ISSUE THREAD

**Issue by kyuusaku** [starimeL/PytorchConverter#14]
Hi, @starimeL .

Do this project support nn.ConvTranspose2d layer?

**Comment by starimeL** [comment]
Yes, the converter support ConvTranspose layer. Just make sure to use the pytorch 0.2.0 to do the conversion.
